src/Allocine/TwigLinter/Rule/AbstractRule.php

```diff
@@ -2,6 +2,7 @@
 
 namespace Allocine\TwigLinter\Rule;
 
+use Allocine\TwigLinter\Token;
 use Allocine\TwigLinter\Validator\Violation;
 
 class AbstractRule
```

Comment:
the class is not declared as abstract, shouldn't it be? There's no phpdoc neither